### PR TITLE
HPCC-28266 Roll back the code for retrieving actual disk space

### PR DESCRIPTION
**Pull request by wangkx**
The code was added by https://github.com/hpcc-systems/HPCC-Platform/pull/15379. The code may 'hang' the service when some folders are nfs mounted and have thousands (or millions) of files.

Also add/fix 2 debug lines for better debug log.

Signed-off-by: wangkx <kevin.wang@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/services/ws_machine/ws_machineService.cpp [hunk 1]
```diff
@@ -2808,8 +2796,6 @@ void Cws_machineEx::readComponentUsageResult(IEspContext& context, IPropertyTree
                             VStringBuffer desc("%s not found. Read disk usage from %s", aDiskPath, pathUsed);
                             diskUsage->setDescription(desc);
                         }
-                        if ((version >= 1.18) && folderTree->hasProp("@size"))
```
Comment (mckellyln) [inline esp/services/ws_machine/ws_machineService.cpp:2811]: Is there any compatibility issue if you remove this ?  Is it safer to just leave the third argument but have it be "0" for now ?

Comment (wangkx) [reply esp/services/ws_machine/ws_machineService.cpp:2811]: The 2 lines of the code sets the ActualSIze only if the folerTree has the '@size'. Theoretically, the client side should not have a problem if the client does not receive the ActualSIze. So, I think it should no compatibility issue here.

**Review by mckellyln — APPROVED** [review]
ok, approved

**Review by afishbeck — APPROVED** [review]

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged.

File: esp/scm/ws_machine.ecm [hunk 2]
```diff
@@ -32,7 +32,6 @@ ESPStruct [nil_remove] DiskUsage
     int64 Available;
     int PercentAvailable;
     [min_ver("1.17")] string Exception;
-    [min_ver("1.18")] int64 ActualSize;
```
Comment (richardkchapman) [inline esp/scm/ws_machine.ecm:35]: Feels like removing a versioned variable may give compatibility issues, no?

**Review by richardkchapman — CHANGES_REQUESTED** [review]

File: esp/scm/ws_machine.ecm (hunk 2 above)
Comment (wangkx) [reply esp/scm/ws_machine.ecm:35]: Without this change, the DiskUsage may not contain the ActualSize variable if ESP cannot set the value. So, the variable may not always be there. I think that removing this variable should not give compatibility issues. If really needed, I can add the code to increase the version and depr this variable. Please let me know.

**Review by richardkchapman — APPROVED** [review]

**Review by ghalliday — APPROVED** [review]

**Pull request merged**